content/tutorial/12-errors.md

```diff
@@ -124,10 +124,10 @@ def setup(bot: commands.Bot):
     bot.add_cog(ErrorHandler(bot))
 ```
 
-Pretty simple, that's essentially just the standard template for a cog as normal. Next we're going to add a listener to receive the command errors:
+Pretty simple, that's essentially just the standard template for a cog as normal. Next, we're going to add a listener to receive the command errors:
 
 ```py
-    # Assume this is all between the class' constructor and the setup method.
+    # Assume this is all between the class constructor and the setup method.
```

Comment:
This was correct as it was, it's referring to the constructor that belongs to the class, as opposed to the `class constructor`